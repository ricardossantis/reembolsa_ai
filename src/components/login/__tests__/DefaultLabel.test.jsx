import DefaultLabel from "../default-label";
import React from "react";
import renderer from "react-test-renderer";

describe("Render Test", () => {
  it("renders DefaultLabel", () => {
    const component = renderer.create(<DefaultLabel/>).toJSON();
    expect(component).toMatchSnapshot();
  });
});

