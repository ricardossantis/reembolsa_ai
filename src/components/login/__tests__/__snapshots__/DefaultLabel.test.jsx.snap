// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Test renders DefaultLabel 1`] = `
<label
  className="sc-bdnylx dKMMFq"
/>
`;
